docs/env-variables.md

```diff
@@ -0,0 +1,16 @@
+# Operator environment variables
+
+Operator behavior can be customized using environment variables in the `minio-operator` deployment, here is a list available environment variables to set:
+
+| Variable Name | Description                                                                                                                                                                  | Possible values         | default                       |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|-------------------------------|
+|MINIO_OPERATOR_CERTIFICATES_VERSION| Is the ENV var to force the certificates api version to use.                                                                                                                 | v1,v1beta1              | whatever api k8s provides     |
+|MINIO_OPERATOR_RUNTIME | Tells us which runtime we have. (EKS, Rancher, OpenShift, etc...), for example if `EKS` is set, the csr signerName to be used will be `"beta.eks.amazonaws.com/app-serving"` | EKS, Rancher, OpenShift |                               |
+|MINIO_OPERATOR_CSR_SIGNER_NAME| Is the name to use for the CSR Signer, will override the default                                                                                                             |                         | kubernetes.io/kubelet-serving |
```

Comment:
```suggestion
|MINIO_OPERATOR_CSR_SIGNER_NAME| The name to use for the CSR Signer. It will override the default                                                                                                             |                         | `kubernetes.io/kubelet-serving` |
```